ruby-challenges.rb

```diff
@@ -10,6 +10,17 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+puts "Please enter a number:"
+ input = Integer(gets.chomp)
+ if input % 2 == 0
+ puts "is even"
+ else
+ puts "is odd"
+ end
```

Comment:
I like that you added the ability for the user to provide an input.  Let's try to set this up as a method next time.